testrail/api.py

```diff
@@ -638,7 +638,7 @@ def _post(self, uri, data={}):
 
     def _payload_gen(self, fields, data):
         payload = dict()
-        for field in fields:
-            if data.get(field) is not None:
-                payload[field] = data.get(field)
+        for k, v in data.items():
+            if v is not None:
+                payload[k] = v
```

Comment:
In case of using custom Testrail fields (e.g. for cases) - these values are filtered. 
So, I need to remove filtering (as in PR) or patch fields list on API methods.